extensions/wikia/MyHome/SpecialWikiActivity.class.php

```diff
@@ -66,13 +66,13 @@ function execute($par) {
 			else {
 				$this->feedSelected = 'watchlist';
 				$feedProxy = new WatchlistFeedAPIProxy();
-				$feedRenderer = new WatchlistFeedRenderer();
+				new WatchlistFeedRenderer();
 			}
 		} else {
 		//for example: wiki-domain.com/wiki/Special:WikiActivity
 			$this->feedSelected = 'activity';
 			$feedProxy = new ActivityFeedAPIProxy();
-			$feedRenderer = new ActivityFeedRenderer();
+			new ActivityFeedRenderer();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Either remove this useless object instantiation of class "ActivityFeedRenderer" or use it [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1848)